app/controllers/api/v1/book_controller.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../../serializer/book_serializer'
+
+module Api
+  module V1
+    class BookController < ApplicationController
+      before_action :authenticate_user!
+      def index
+        @books = Book.all
+        render_paginated json: Panko::Response.new(
```

Comment:
Read the config in wor-paginated https://github.com/Wolox/wor-paginate#working-with-panko-serializer